.github/workflows/ci.yml

```diff
@@ -45,7 +45,6 @@ jobs:
       python-version: ${{ matrix.python-version }}
 
   test-platform-compat:
-    if: github.event_name == 'push'
```

Comment:
Why is this being removed?

Reply:
Temporary, just to get the windows compat CI to run - it has been reverted in latest commit.

Reply:
Interesting! Cheers.